docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,228 @@
+---
+title: ConvertTimezone
+---
+
+# ConvertTimezone
+
+`convertTimezone` shifts a timestamp into a specified time zone by adding or subtracting the right interval from the timestamp.
+
+| Syntax                                                                | Example                                                                           |
+|-----------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| `convertTimezone(column, target, source)`                             | `convertTimezone("2022-12-28T12:00:00", "PST", "EST")`                            |
+| Shifts a timestamp from the source time zone to the target time zone. | Returns the value `2022-12-28T09:00:00`, displayed as `December 28, 2022 9:00 AM` |
+
+Timestamps and time zones are rather nasty to work with (it's easy to make mistakes, and difficult to catch them), so you should only try to use `convertTimezone` if the interpretation of your data is very sensitive to time-based cutoffs.
+
+For example, if you're tracking user logins over time, you probably won't run your business differently if some logins get counted on Mondays instead of Tuesdays. However, if you're using Metabase to do something precise, like your taxes, you (and the government) will probably care a lot more about the difference between transactions that occurred on Dec 31 vs. Jan 1.
+
+## Parameters
+
+`column` can be any of:
+- The name of a timestamp column,
+- a custom expression that returns a [timestamp](#accepted-data-types), or
+- a string in the format `"YYYY-MM-DD` or `"YYYY-MM-DDTHH:MM:SS"`.
+
+`target`:
+- The name of the time zone you want to assign to your column.
+- Time zone names are database-dependent (such as "Canada/Eastern" vs. "EST").
+
+`source`:
+- Optional for columns or expressions with the data type `timestamp with time zone`. 
+- Required for columns or expressions with the data type `timestamp without time zone`. 
+- For more info, see [Accepted data types](#accepted-data-types).
+
+## Creating custom report dates
+
+Let's say that you have some time series data that's stored in one or more time zones (**Source Time**). You want to create custom reporting dates for a team that lives in EST.
+
+| Source Time                        | Team Report Time (EST)       |
+|------------------------------------|------------------------------|
+| December 28, 2022, 10:00:00        | December 28, 2022, 07:00:00  |
+| December 28, 2022, 21:00:00        | December 28, 2022, 19:00:00  |
+| December 27, 2022, 08:00:00        | December 27, 2022, 05:00:00  |
+
+If **Source Time** is stored as a `timestamp with time zone` or a `timestamp with offset`, you only need to provide the `target` time zone:
+
+```
+convertTimezone([Source Time], 'EST')
+```
+
+If **Source Time** is stored as a `timestamp without time zone`, you _must_ provide the `source` time zone as 'UTC':
```

Comment:
```suggestion
If **Source Time** is stored as a `timestamp without time zone`, you _must_ provide the `source` time zone ( in this case it is "UTC" ):
```

I feel like the combination of `must`  + a specific timezone could give the wrong idea that it has to be `UTC` all the time?

Reply:
@qnkhuat Yes, I remember being confused about this! I wasn't sure if `timestamp without time zone` would get treated as UTC by default.

What about this:
> If **Source Time** is stored as a `timestamp without time zone`, you _must_ provide the `source` time zone (which will depend on your database time zone):